### DISCUSSION TITLE
Pick up system reserved threshold 500ms because of config missing

### DISCUSSION THREAD

**Discussion by joenzx** [apache/skywalking#8117] [General]
Can someone please help to check which file is wrong?

Which configuration file is missing?

![image](https://user-images.githubusercontent.com/42016493/141446309-9c9ef13f-ea40-4436-8633-ba10a79becdd.png)

**Comment by wu-sheng** [comment]
I think the new version has fixed this, and you don't need to worry about this.

**Reply by joenzx** [reply]
I see service-apdex-threshold-yml in config.
I'm using 8.6, but the log output is still fast。

**Reply by wu-sheng** [reply]
I think this is just warning, not an error. 
File name is `service-apdex-threshold.yml`, and you at least should have a `default`. 
This could be caused by no legal config file, or you activated dynamic configuration, but don't provide the correct configuration.
Anyway, it if better for you to recheck locally.

**Reply by danishlyy** [reply]
> I think this is just warning, not an error. File name is `service-apdex-threshold.yml`, and you at least should have a `default`. This could be caused by no legal config file, or you activated dynamic configuration, but don't provide the correct configuration. Anyway, it if better for you to recheck locally.

hello I'm using 8.7.0, it appears the same message, I'm using nacos as dynamic configuration, adding service-apdex-threshold.yml on it, restart oap and webapp, the ui still cannot see logs in log module,what should I do to solve this. thanks
